admin-dev/themes/new-theme/js/pages/order/create/event-map.js

```diff
@@ -31,6 +31,8 @@ export default {
   customerSearched: 'customerSearched',
   // when new customer is selected
   customerSelected: 'customerSelected',
+  // when no customers found by search
+  customersNotFound: 'customersNotFound',
```

Comment:
```suggestion
  searchCustomerNotFound: 'OrderCreateSearchCustomerNotFound',
```

Event must have the context in the key. 
Otherwise we will have the same event name in the Customer Page when trying to edit the customer.